bin/generate_jobs_yaml.sh

```diff
@@ -13,147 +13,241 @@ cd "$REPO"
 
 # Here are the supported nginx tags.
 version_table=$(mktemp)
-# base-image    nginx-version    nginx-modules-path    nginx-conf-path    always-build?
```

Comment:
I had it so that commits run only on the most recent versions of the supported images, to save time in cycles. Then what you call `release_tag_only` was for releases.

Do you think that it's better to run everything all the time, or is this an artifact of getting ARM to work?

Or maybe since you simplified things by adding `release_tag_only`, the idea is that we can manually remove that line in the first few sections whenever we update the YAML.